### PR TITLE
DX-2192 Update Voice API Spec

### PR DESCRIPTION
**Pull request by ajrice6713**
Update Tags, Summaries, and Descriptions of each endpoint to better display the reference on the docsite

## For the Committer

All PRs on this repo that change any API sources of truth (markdown files, OpenAPI specs) require a changelog added to the `site/src/pages/changelog.md` file. If this PR does not require changelog updates, you need to add the `no-changelog` tag to this PR before opening it.

Please confirm that you have either updated `site/src/pages/changelog.md` or added the `no-changelog` tag.

### CODE REVIEW

File: site/specs-source/voice.json [hunk 1]
```diff
@@ -504,9 +504,9 @@
     "/api/v2/accounts/{accountId}/calls/{callId}/recordings": {
       "get": {
         "tags": [
-          "recordings"
+          "Recordings"
         ],
-        "summary": "Get metadata for all of the recordings that took place during the specified call.",
+        "summary": "Get Call Recordings",
         "description": "Returns a (potentially empty) list of metadata for the recordings that took place during the specified call",
```
Comment (hamermike) [inline site/specs-source/voice.json:510]:
```suggestion
        "description": "Returns a (potentially empty) list of metadata for the recordings that took place during the specified call.",
```

**Review by hamermike — CHANGES_REQUESTED** [review]

File: site/specs-source/voice.json [hunk 2]
```diff
@@ -2413,9 +2413,9 @@
     "/api/v2/accounts/{accountId}/recordings": {
       "get": {
         "tags": [
-          "recordings"
+          "Recordings"
         ],
-        "summary": "Returns metadata for up to 1000 of your account's most recent recordings (optionally filterable).",
+        "summary": "Get Recordings",
```
Comment (abaldwin-Bandwidth) [inline site/specs-source/voice.json:2418]:
```suggestion
        "summary": "Get Call Recordings",
```

File: site/specs-source/voice.json [hunk 3]
```diff
@@ -618,10 +618,10 @@
     "/api/v2/accounts/{accountId}/calls/{callId}/recordings/{recordingId}": {
       "get": {
         "tags": [
-          "recordings"
+          "Recordings"
         ],
-        "summary": "Returns metadata for the specified recording.",
-        "description": "",
+        "summary": "Get Recording Information",
```
Comment (abaldwin-Bandwidth) [inline site/specs-source/voice.json:623]:
```suggestion
        "summary": "Get Call Recording",
```

**Review by abaldwin-Bandwidth — CHANGES_REQUESTED** [review]
Recordings API call name verbiage was awkward

**Review by hamermike — APPROVED** [review]

**Review by abaldwin-Bandwidth — APPROVED** [review]

**Pull request merged**